model/minsky.cc

```diff
@@ -264,6 +265,35 @@ namespace minsky
     GroupPtr g(new Group);
     canvas.setItemFocus(model->addGroup(g));
     m.populateGroup(*g);
+    g->resizeOnContents();	  
+  }
+
+    // Fix for ticket 1080
+  void Minsky::pasteItems()
+  {
+    istringstream is(getClipboard());
+    xml_unpack_t unpacker(is);
+    schema2::Minsky m;
+    xml_unpack(unpacker, "Minsky", m);
+    GroupPtr g(new Group);
+    canvas.setItemFocus(model->addGroup(g));
```

Comment:
You shouldn't be adding the group to the model, as you will be unpacking it later.